streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/SubscriptionInfoTest.java

```diff
@@ -173,18 +177,19 @@ public void generatedVersion2ShouldDecodeLegacyFormat() {
             2,
             LATEST_SUPPORTED_VERSION,
             processId,
-            activeTasks,
-            standbyTasks,
-            "localhost:80"
+            ACTIVE_TASKS,
+            STANDBY_TASKS,
+            "localhost:80",
+            null
```

Comment:
Q: Why do you pass in `null` here but in some other cases you pass in `TASK_LAGS`?

Reply:
Ended up removing the `taskLags`(now `taskOffsetSums`) from the constructor entirely

Reply:
Actually, @vvcephei I have to say I'm not sure I entirely understand the point of this legacy subscription serde. Shouldn't `SubscriptionInfo` also be able to decode older "legacy"versions?

Reply:
Yes, it should... I asked a question about this on that class itself. It seems like you shouldn't have had to modify it at all.